### PR TITLE
plasma-infra: Improve "Cypress" workflow

### PR DESCRIPTION
**Pull request by Yakutoc**
Продолжаем ускорять. 

- **Исключили** scope "temple" из общего сценария для  "Cypress" workflow. 

- Вынесли в отдельный набор jobs который запускается **только** когда внесли изменения в пакет "plasma-temple". 

```yml
on:
  schedule:
    - cron: 0 0 * * SUN
```

- **Добавили** запуск **нового** workflow по расписанию. Теперь каждое воскресение в полночь будет запускаться cypress для всех возможных scope.     

#### Изменения в `plasma-core` запускает тесты только для пакетов b2c, web, ui.

Screenshot 2023-07-04 at 15 33 09

#### Изменения в `plasma-temple` запустили проверку только для этого пакета. Общий сценарий - **skip**.  

Screenshot 2023-07-04 at 16 47 36

----------

:baby_chick: Download canary assets:

[color_asdk_ios-swift--canary.592.5471391911.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_asdk_ios-swift--canary.592.5471391911.swift)  
[color_asdk_kotlin--canary.592.5471391911.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_asdk_kotlin--canary.592.5471391911.kt)  
[color_asdk_react-native--canary.592.5471391911.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_asdk_react-native--canary.592.5471391911.ts)  
[color_asdk_xml--canary.592.5471391911.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_asdk_xml--canary.592.5471391911.xml)  
[color_plasma_b2c_ios-swift--canary.592.5471391911.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_ios-swift--canary.592.5471391911.swift)  
[color_plasma_b2c_kotlin--canary.592.5471391911.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_kotlin--canary.592.5471391911.kt)  
[color_plasma_b2c_react-native--canary.592.5471391911.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_react-native--canary.592.5471391911.ts)  
[color_plasma_b2c_xml--canary.592.5471391911.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_xml--canary.592.5471391911.xml)  
[color_sberHealth_ios-swift--canary.592.5471391911.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.592.5471391911.swift)  
[color_sberHealth_kotlin--canary.592.5471391911.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.592.5471391911.kt)  
[color_sberHealth_react-native--canary.592.5471391911.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.592.5471391911.ts)  
[color_sberHealth_xml--canary.592.5471391911.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_xml--canary.592.5471391911.xml)  
[color_sbermarket_business_ios-swift--canary.592.5471391911.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.592.5471391911.swift)  
[color_sbermarket_business_kotlin--canary.592.5471391911.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.592.5471391911.kt)  
[color_sbermarket_business_react-native--canary.592.5471391911.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.592.5471391911.ts)  
[color_sbermarket_business_xml--canary.592.5471391911.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_xml--canary.592.5471391911.xml)  
[color_sbermarket_ios-swift--canary.592.5471391911.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.592.5471391911.swift)  
[color_sbermarket_kotlin--canary.592.5471391911.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.592.5471391911.kt)  
[color_sbermarket_metro_ios-swift--canary.592.5471391911.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.592.5471391911.swift)  
[color_sbermarket_metro_kotlin--canary.592.5471391911.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.592.5471391911.kt)  
[color_sbermarket_metro_react-native--canary.592.5471391911.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.592.5471391911.ts)  
[color_sbermarket_metro_xml--canary.592.5471391911.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_xml--canary.592.5471391911.xml)  
[color_sbermarket_react-native--canary.592.5471391911.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.592.5471391911.ts)  
[color_sbermarket_selgros_ios-swift--canary.592.5471391911.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.592.5471391911.swift)  
[color_sbermarket_selgros_kotlin--canary.592.5471391911.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.592.5471391911.kt)  
[color_sbermarket_selgros_react-native--canary.592.5471391911.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.592.5471391911.ts)  
[color_sbermarket_selgros_xml--canary.592.5471391911.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_xml--canary.592.5471391911.xml)  
[color_sbermarket_wlbusiness_ios-swift--canary.592.5471391911.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.592.5471391911.swift)  
[color_sbermarket_wlbusiness_kotlin--canary.592.5471391911.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.592.5471391911.kt)  
[color_sbermarket_wlbusiness_react-native--canary.592.5471391911.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.592.5471391911.ts)  
[color_sbermarket_wlbusiness_xml--canary.592.5471391911.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_xml--canary.592.5471391911.xml)  
[color_sbermarket_xml--canary.592.5471391911.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_xml--canary.592.5471391911.xml)  
[color_sberonline_ios-swift--canary.592.5471391911.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_ios-swift--canary.592.5471391911.swift)  
[color_sberonline_kotlin--canary.592.5471391911.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_kotlin--canary.592.5471391911.kt)  
[color_sberonline_react-native--canary.592.5471391911.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_react-native--canary.592.5471391911.ts)  
[color_sberonline_xml--canary.592.5471391911.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_xml--canary.592.5471391911.xml)  
[color_sberprime_ios-swift--canary.592.5471391911.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.592.5471391911.swift)  
[color_sberprime_kotlin--canary.592.5471391911.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.592.5471391911.kt)  
[color_sberprime_react-native--canary.592.5471391911.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.592.5471391911.ts)  
[color_sberprime_xml--canary.592.5471391911.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_xml--canary.592.5471391911.xml)  
[shadow_sbermarket_react-native--canary.592.5471391911.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.592.5471391911.ts)  
[typo_mage_ios-swift--canary.592.5471391911.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.592.5471391911.swift)  
[typo_mage_kotlin--canary.592.5471391911.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.592.5471391911.kt)  
[typo_mage_react-native--canary.592.5471391911.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.592.5471391911.ts)  
[typo_plasma_ios-swift--canary.592.5471391911.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.592.5471391911.swift)  
[typo_plasma_kotlin--canary.592.5471391911.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.592.5471391911.kt)  
[typo_plasma_react-native--canary.592.5471391911.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.592.5471391911.ts)  
[typo_ruler_ios-swift--canary.592.5471391911.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.592.5471391911.swift)  
[typo_ruler_kotlin--canary.592.5471391911.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.592.5471391911.kt)  
[typo_ruler_react-native--canary.592.5471391911.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.592.5471391911.ts)  
[typo_sage_ios-swift--canary.592.5471391911.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.592.5471391911.swift)  
[typo_sage_kotlin--canary.592.5471391911.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.592.5471391911.kt)  
[typo_sage_react-native--canary.592.5471391911.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.592.5471391911.ts)  
[typo_sbermarket_ios-swift--canary.592.5471391911.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.592.5471391911.swift)  
[typo_sbermarket_kotlin--canary.592.5471391911.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.592.5471391911.kt)  
[typo_sbermarket_react-native--canary.592.5471391911.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.592.5471391911.ts)  
[typo_soulmate_ios-swift--canary.592.5471391911.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.592.5471391911.swift)  
[typo_soulmate_kotlin--canary.592.5471391911.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.592.5471391911.kt)  
[typo_soulmate_react-native--canary.592.5471391911.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.592.5471391911.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.218.1-canary.592.5471391911.0
  npm install @salutejs/plasma-core@1.123.1-canary.592.5471391911.0
  npm install @salutejs/plasma-hope@1.218.1-canary.592.5471391911.0
  npm install @salutejs/plasma-icons@1.152.1-canary.592.5471391911.0
  npm install @salutejs/plasma-temple@1.167.1-canary.592.5471391911.0
  npm install @salutejs/plasma-ui@1.199.1-canary.592.5471391911.0
  npm install @salutejs/plasma-web@1.218.1-canary.592.5471391911.0
  npm install @salutejs/plasma-sb-utils@0.121.1-canary.592.5471391911.0
  # or 
  yarn add @salutejs/plasma-b2c@1.218.1-canary.592.5471391911.0
  yarn add @salutejs/plasma-core@1.123.1-canary.592.5471391911.0
  yarn add @salutejs/plasma-hope@1.218.1-canary.592.5471391911.0
  yarn add @salutejs/plasma-icons@1.152.1-canary.592.5471391911.0
  yarn add @salutejs/plasma-temple@1.167.1-canary.592.5471391911.0
  yarn add @salutejs/plasma-ui@1.199.1-canary.592.5471391911.0
  yarn add @salutejs/plasma-web@1.218.1-canary.592.5471391911.0
  yarn add @salutejs/plasma-sb-utils@0.121.1-canary.592.5471391911.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-592/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-592/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-592/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-592/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-592/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-592/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-592/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-592/

File: .github/workflows/cypress-by-cron.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: Cypress component testing by schedule
+
+on:
+  schedule:
+    - cron: 0 0 * * SUN
+
+jobs:
+  cypress:
+    strategy:
+      fail-fast: false
```
Comment (Yeti-or) [inline .github/workflows/cypress-by-cron.yml:10]: я забыл это зачем нужно ?

**Review by Yeti-or — APPROVED** [review]

**Review by kayman233 — APPROVED** [review]

File: .github/workflows/cypress-by-cron.yml (hunk 1 above)
Comment (Yakutoc) [reply .github/workflows/cypress-by-cron.yml:10]: @Yeti-or 

TL;DR - Что бы остальные job продолжали работу, если какая-нибудь вдруг отвалилась.  

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast

> **fail-fast** applies to the entire matrix. If **fail-fast** is set to true, GitHub will cancel all in-progress and queued jobs in the matrix if any job in the matrix fails. This property defaults to true.

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-592/

**Review by neretin-trike — APPROVED** [review]

**Pull request merged**